config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Hubstats::Engine.routes.draw do
   root to: "pull_requests#index"
 
-  post "/handler" => "events#handler", :as => "handler"
-  get "/metrics" => "repos#dashboard", :as => "metrics" 
+  post "/handler" => "events#handler", :as => :handler
+  post "/deploy" => "deploys#create"
```

Comment:
You could take advantage of Rails resource routing conventions here and reduce your deploy routes to one line. Something like this should work:
{code}
resources :deploys, :only => [:index, :create]
{code}

Details: http://guides.rubyonrails.org/v3.2.21/routing.html#crud-verbs-and-actions